pyphare/pyphare/pharesee/run/utils.py

```
@@ -359,16 +358,16 @@ def _get_rank(patchdatas, patch_id, **kwargs):
     layout = reference_pd.layout
     centering = "dual"
     nbrGhosts = layout.nbrGhosts(layout.interp_order, centering)
-    shape = grow(reference_pd.box, [nbrGhosts] * 2).shape
+    shape = grow(reference_pd.box, [nbrGhosts] * ndim).shape
 
     if ndim == 1:
-        pass
+        raise RuntimeError("Not used in 1D")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)